browser/src/control/Control.Tabs.js

```diff
@@ -307,7 +315,7 @@ L.Control.Tabs = L.Control.extend({
 	},
 
 	_setPart: function (e) {
-		var part =  e.target.id.match(/\d+/g)[0];
+		var part =  e.currentTarget.id.match(/\d+/g)[0];
```

Comment:
this is need because we want the element the handler is attached to (bubbling) instead of the inside elements.